### PR TITLE
[FrameworkBundle] Simplified the MicroKernelTrait setup

### LINKED ISSUE #19975: [FrameworkBundle] Simpler Kernel setup with `MicroKernelTrait`
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#57408
| PR author(s) | @yceruto
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #19975.

### CODE REVIEW

**Comment by yceruto** [comment]
Javier thanks for opening this PR so quickly :) 

I was wondering if we could simplify the example even more by referencing the runtime autoload file instead and replace this code:
```php
$kernel = new Kernel('dev', true);
$request = Request::createFromGlobals();
$response = $kernel->handle($request);
$response->send();
$kernel->terminate($request, $response);
```
with this other:
```php
return static function (array $context) {
    return new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']);
}
```
wdyt?

**Comment by javiereguiluz** [comment]
I like your proposal Yonel. I updated the PR with your changes. Thanks for the review.

File: configuration/micro_kernel_trait.rst [hunk 1]
```diff
@@ -34,19 +34,12 @@ Next, create an ``index.php`` file that defines the kernel class and runs it:
         use Symfony\Component\HttpKernel\Kernel as BaseKernel;
         use Symfony\Component\Routing\Attribute\Route;
 
-        require __DIR__.'/vendor/autoload.php';
+        require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
```
Comment (yceruto) [inline configuration/micro_kernel_trait.rst:35]: Don't forget to update the `composer require` section with `symfony/runtime` libs ;)

Comment (yceruto) [reply configuration/micro_kernel_trait.rst:35]: In that section only `symfony/framework-bundle` is required actually, the other ones are transitive dependencies anyway, not sure we wanted to make them explicit though.

Comment (javiereguiluz) [reply configuration/micro_kernel_trait.rst:35]: Good catch! I think that, generally speaking, we don't list transitive dependencies in the docs, so let's remove anything that is not strictly needed. Thanks!

**Review by yceruto — APPROVED** [review]

**Pull request merged**